src/Sarif/HashUtilities.cs

```diff
@@ -20,7 +20,7 @@ internal static IFileSystem FileSystem
         {
             get
             {
-                _fileSystem = _fileSystem ?? new FileSystem();
+                _fileSystem ??= new FileSystem();
```

Comment:
>new FileSystem() [](start = 32, length = 16)

`FileSystem.Instance`

Reply:
this is complicated...since this class has FileSystem property, i couldn't replace with FileSystem.Instance. Should  I rename the property?

---
In reply to: [505677459](https://github.com/microsoft/sarif-sdk/pull/2115#discussion_r505677459) [](ancestors = 505677459)

Reply:
You can namespace it: `_fileSystem ??= Microsoft.CodeAnalysis.FileSystem.Instance;`

---
In reply to: [505680343](https://github.com/microsoft/sarif-sdk/pull/2115#discussion_r505680343) [](ancestors = 505680343,505677459)

Reply:
I would have removed this property and just marked the underlying private static as 'internal'. :)

---
In reply to: [505681908](https://github.com/microsoft/sarif-sdk/pull/2115#discussion_r505681908) [](ancestors = 505681908,505680343,505677459)